cmd/kubevirt-csi-driver/kubevirt-csi-driver.go

```diff
@@ -60,7 +58,6 @@ func init() {
 
 func main() {
 	flag.Parse()
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Because it is not needed anymore, rand.Seed is deprecated, and according to the documentation if you don't initialize the Random with a particular seed, it gets a random seed on startup.

Reply:
Okay so the automatic seed population doesn't happen automatically until golang 1.20, I will restore this until we update golang to be 1.20